cmd/vitconfig/vitconfig.go

```diff
@@ -1027,7 +1028,7 @@ func main() {
 		kit.FatalOn(err, "vcli.DbInit", kit.B2S(out))
 
 		// populate the database with already dumped data
-		out, err = vcli.CsvDataLoad(vitDb, fundsFile.Name(), proposalsFile.Name(), votePlansFile.Name())
+		out, err = vcli.CsvDataLoad(vitDb, fundsFile.Name(), proposalsFile.Name(), *challengesPath, votePlansFile.Name())
```

Comment:
One of these is not like the others. I guess `.Name()` does some useful normalization on paths, so it's better used for the challenges file as well?

Reply:
It is different, the others are actually modified by the script itself, and dumped into another file (those file objects are the ones used to get the new path there with the `.Name()` method).  But in the case of challenges, we just bridge it to the cli itself. It feels a bit overwork opening a file just to use the same syntax.